scipy/meson.build

```diff
@@ -67,6 +68,15 @@ lapack = dependency('openblas')
 #       _build_utils/_fortran.py)
 g77_abi_wrappers = files('_build_utils' / 'src' / 'wrap_dummy_g77_abi.f')
 
+python_exe = find_program('python3')
+config = files('../tools/config_utils.py')
+version = files('../tools/version_utils.py')
+generate_config = run_command(python_exe, config, '__config__.py')
+generate_version = run_command(python_exe, version)
+
+if generate_config.returncode() != 0 or generate_version.returncode() !=0
+    error('Error message')
```

Comment:
This needs a more descriptive error message.

Reply:
Added a more descriptive error message.